runing/repositories.py

```
@@ -0,0 +1,54 @@
+
+    def delete_user(slef, user_id: int) -> bool:
+        db_user = self.get_user(user_id)
+        if db_user:
+            update_data = user_update.dict(exclude_unset=True)
+            for field. value in update_data.items():
+                setattr(db_user, field, value)
+            self.db.commit()
+            self.db.refresh(db_user)
+        return db_user
+
+    def delete_user(delf, user_id: int) -> bool:
```

Ruff (0.11.9)

Redefinition of unused `delete_user` from line 37

(F811)